numerics/polynomial_evaluators_body.hpp

```diff
@@ -0,0 +1,221 @@
+#pragma once
+
+#include "numerics/polynomial_evaluators.hpp"
+
+#include <tuple>
+
+namespace principia {
+namespace numerics {
+namespace internal_polynomial_evaluators {
+
+namespace {
+
+// Greatest power of 2 less than or equal to n.  8 -> 8, 7 -> 4.
+constexpr int flp2(int const n) {
+  return n == 0 ? 0 : n == 1 ? 1 : flp2(n >> 1) << 1;
+}
+
+// Ceiling log2 of n.  8 -> 3, 7 -> 2.
+constexpr int log2(int const n) {
```

Comment:
Why not explicitly CeilingLog2? also why the styleguide-violating lowercase?

Reply:
Done.